pipe_anchorages/port_visits_pipeline.py

```diff
@@ -96,44 +77,32 @@ def drop_new_fields(x):
     excluded_fields = {"ssvid", "duration_hrs", "confidence"}
     return {key: value for key, value in x.items() if key not in excluded_fields}
 
-strdate_to_utcdate = lambda strdate: datetime.datetime.strptime(strdate, "%Y-%m-%d").replace(tzinfo=pytz.utc)
+
+def strdate_to_utcdate(strdate):
+    return datetime.datetime.strptime(strdate, "%Y-%m-%d").replace(tzinfo=pytz.utc)
+
 
 def run(options):
     visit_args = options.view_as(PortVisitsOptions)
     cloud_args = options.view_as(GoogleCloudOptions)
 
     config = cmn.load_config(visit_args.config)
 
-    p = beam.Pipeline(options=options)
+    pipeline = beam.Pipeline(options=options)
 
     start_date = strdate_to_utcdate(visit_args.start_date)
     end_date = strdate_to_utcdate(visit_args.end_date)
 
-    anchorages = (
-        p
-        | "ReadAnchorages"
```

Comment:
@bitsofbits ohh, I saw that in the DF graph it wasn't linked, so.. no more need to read anchorages when generating the port visits, is it ok if we also remove if from [options](https://github.com/GlobalFishingWatch/anchorages_pipeline/blob/release-v4/pipe_anchorages/options/port_visits_options.py#L27)?

Reply:
@smpiano , good idea. I just pulled that out, which also meant removing it from the VisitSink class where it was also being referenced (unnecessarily).